### DISCUSSION TITLE
osu game rulesets

### DISCUSSION THREAD

**Discussion by crse** [opengaming/osgameclones#2087] [Q&A] [answered]
So, [osu](https://github.com/ppy/osu), the most popular open source rhythm games, is allowing third party games rulesets. Those rulesets usually a clone to other existing games. These ruleset are basically dependent on osu itself, **not** a separate software. Most of these rulesets are open source.

Ruleset examples:
- Hishigata (clone of [Project Rhombus](https://store.steampowered.com/app/735570/Project_Rhombus/))
source: https://github.com/LumpBloom7/hishigata
- rush (clone of [Muse Dash](https://github.com/Beamographic/rush))
source: https://github.com/Beamographic/rush
- Hitokori (clone of [A Dance of Fire and Ice](https://store.steampowered.com/app/977950/A_Dance_of_Fire_and_Ice/))
source: https://github.com/Flutterish/Hitokori
- osu-DIVA (clone of [Hatsune Miku: Project Diva](https://en.wikipedia.org/wiki/Hatsune_Miku:_Project_DIVA))
source: https://github.com/Artemis-chan/osu-DIVA
- sentakki (clone of Sega's [Maimai](https://en.wikipedia.org/wiki/Maimai_(video_game_series)) arcade machine)
source: https://github.com/LumpBloom7/sentakki

I want to add them to the database, but they're not exactly match to existing type categorization.

Should I just add them as `clone`? Or maybe it requires unique type like `add-on`?

**Comment by cxong** [comment, marked as answer]
Clone is fine, with osu as a framework. See OpenKrush as an example, which uses OpenRA: https://osgameclones.com/krush-kill-n-destroy/

**Reply by crse** [reply]
Thanks for the confirmation. I'll do exactly that :)
